backend/authschemes/auth_bridge_test.go

```diff
@@ -59,56 +60,50 @@ func TestCreateNewUser(t *testing.T) {
 
 type testSession struct{ Some string }
 
+func createContextForRequest(sessionManager scs.SessionManager, r *http.Request, w http.ResponseWriter) *http.Request {
+	var token string
+	cookie, err := r.Cookie("cookie-name")
+	if err == nil {
+		token = cookie.Value
+	}
+
+	ctx, err := sessionManager.Load(r.Context(), token)
+	if err != nil {
+		sessionManager.ErrorFunc(w, r, err)
+		return r
+	}
+	return r.WithContext(ctx)
+}
+
 func TestLoginUser(t *testing.T) {
-	_, sessionStore, bridge := initBridgeTest(t)
+	_, sessionManager, bridge := initBridgeTest(t)
 
 	gob.Register(&testSession{})
 
 	userID := createDummyUser(t, bridge, "")
 
 	browser := &testBrowser{}
 	w, r := browser.newRequest()
-	err := bridge.LoginUser(w, r, userID, &testSession{Some: "data"})
+	sr := createContextForRequest(*sessionManager, r, w)
+	err := bridge.LoginUser(w, sr, userID, &testSession{Some: "data"})
```

Comment:
Note that in this file we're not actually seeing if the middleware works correctly (which is why we use the same request twice instead of creating a new request), but just ensuring that the bridge functions (`bridge.LoginUser`, `bridge.SetAuthSchemeSession` etc) don't error out, since they're basically wrappers for the SCS api